dts/bindings/sensor/st,lis3dh-i2c.yaml

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2018 STMicroelectronics
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: STMicroelectronics MEMS sensors LIS3DH
+version: 0.1
+
+description: >
+    This binding gives a base representation of LIS3DH 3-axis accelerometer
+
+inherits:
+    !include i2c-device.yaml
+
+properties:
+    compatible:
+        constraint: "st,lis3dh-i2c"
```

Comment:
Since we only support the i2c mode right now, lets just call this "st,lis3dh"

Reply:
Actually I did it in this way because we may want to support spi as well soon.

Reply:
But maybe not that soon.

Reply:
ok, repushed